.github/workflows/codeforafrica-deploy-review-app.yml

```diff
@@ -92,7 +92,7 @@ jobs:
   destroy_review_app:
     runs-on: ubuntu-latest
     # only run when a pull request is closed
-    if: github.event_name == 'pull_request' && github.event.action == 'closed'
+    if: github.event_name == 'pull_request' && (github.event.action == 'closed' || github.event.pull_request.merged == true)
```

Comment:
Doesn't `closed` include `merged`?

PS: Lets deal with this in a separate PR if need be.